cdk/dcm_floodlight/config.yaml

```diff
@@ -1,11 +1,23 @@
 message:
   supportedMessageTypes:
     - track
+    - page
   track:
     transformation:
       mapperPath: ./mapping/track.yaml
       postMapper:
-        name: "trackPostMapper"
+        name: "postMapper"
+      response:
+        endpoint: "{{ rudderContext.endpoint }}"
+        method: GET
+        format: JSON
+        headers:
+          User-Agent: "{{ rudderContext.userAgent }}"
+  page:
+    transformation:
+      mapperPath: ./mapping/track.yaml
```

Comment:
`track.yaml` should be change to something genric name

Reply:
changed to DCMFloodlightConfig